bin/update-ips.bash

```diff
@@ -140,10 +151,27 @@ checkIfDiffAndUpdateKubectlIPs() {
     fi
 }
 
-# checkIfDiffAndUpdatePorts <filepath> <yamlpath> <ports>...
+# checkIfDiffAndUpdateIPs <yaml_path> <file> <IPs> ...
+checkIfDiffAndUpdateIPs() {
+    local yaml_path="${1}"
+    local file="${2}"
+    shift 2
+    local IPS=("$@")
+
+    if ! diffIPs "${yaml_path}" "${file}" "${IPS[@]}"; then
+        if ! $DRY_RUN; then
+            updateIPs "${yaml_path}" "${file}" "${IPS[@]}"
+        else
+            log_warning "Diff found for ${yaml_path} in ${file//${CK8S_CONFIG_PATH}\//} (diff shows actions needed to be up to date)"
+        fi
+        has_diff=$(( has_diff + 1 ))
+    fi
+}
+
+# checkIfDiffAndUpdatePorts <yamlpath> <filepath> <ports>...
 checkIfDiffAndUpdatePorts() {
```

Comment:
The other functions are flipped the other way around. Could you keep that?
So <something> <yamlpath>

Reply:
I'm not sure If I want to change anything more, it was only the `checkIfDiffAndUpdatePorts` that had flipped the yaml_path and file arguments.
I think it is fine to have `<something> <yaml_path> <file> <something_else>`.

Reply:
OK with this @robinAwallace ?

Reply:
Yes this looks good :+1: